src/modals/TransferModal.js

```diff
@@ -378,20 +397,38 @@ class TransferModal extends React.Component {
   };
 
   onToAddressChange = async (e) => {
-    const address = e.target.value;
+    const value = e.target.value;
+    let address = value;
+    if (value.toLowerCase().endsWith(".eth")) {
```

Comment:
Isn't there any other generic validation to check for ENS addresses?

There are many Argent users in DeFi and their addresses end with `.xyz`. See this example:

https://etherscan.io/address/ebirbe.argent.xyz

Reply:
Please create an issue or a PR. Thanks.

Reply:
Opened PR at #160